app/controllers/todos_controller.rb

```diff
@@ -1,18 +1,23 @@
 class TodosController < ApplicationController
   def index
-    todos = GetTodosService.run(
-      :per => params[:per],
-      :page => params[:page],
-      :sorting_criteria => params[:sorting_criteria],
-      :order => params[:order],
-      :search_string => params[:search_string],
-      :filter_criteria => params[:filter_criteria]
-    )
+    case
+      when params[:all_todos_ids] then render :json => GetAllTodosIdService.run!()
+      when params[:id] then render :json => GetTodoByIdService.run!(:id => params[:id])
+      else
```

Comment:
default?

Reply:
Isn't `else` the equivalent for `default` in Ruby?

Reply:
yes, sorry :facepalm:  my bad